src/Network/NWWebSocketMetadata.cs

```diff
@@ -0,0 +1,118 @@
+//
+// NWWebSocketMetadata.cs: Bindings the Network nw_browser_t API.
+//
+// Authors:
+//   Manuel de la Pena (mandel@microsoft.com)
+//
+// Copyrigh 2019 Microsoft Inc
+//
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using ObjCRuntime;
+using Foundation;
+using CoreFoundation;
+
+using OS_nw_protocol_metadata=System.IntPtr;
+using OS_nw_ws_response=System.IntPtr;
+using dispatch_queue_t =System.IntPtr;
+
+namespace Network {
+
+	[TV (13,0), Mac (10,15), iOS (13,0), Watch (6,0)]
+	public enum NWWebSocketOpCode : long {
```

Comment:
`nw_ws_opcode_t` doesn't have a specific type in the headers, which means it's an `int`, not `long`.